apps/publishers/tests/portal/members/test_publisher_member_acceptance_criteria.py

```
@@ -58,7 +65,7 @@ def test_invite_creates_pending_and_emails(self):
 
     # AC#6 + AC#2: accept → active member + password set + token unusable. Staff get READ baseline only.
     def test_accept_makes_active_read_baseline_for_staff_and_token_single_use(self):
-        member, inv, raw = self._invite_via_service(self.p1, "e2@example.com", role="staff")
+        member, inv, raw = self._invite_via_service(self.p1, "e2@example.com", group=member_group())
```

Ruff (0.16.1)

[warning] Unpacked variable `inv` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)